impress-org/givewp
# Fix/ Donor wall now displays donation dates

Resolves #6466

## Description

The updated Donor Wall Block for 2.21.0 is missing the date field on the front end.
In the previous version, this date field was available. 

## Affects

Donor Wall.

## Visuals

https://user-images.githubusercontent.com/75056371/174670044-7e1ecf69-ed71-4f6b-a8d2-cb827fa5dc96.mov


## Testing Instructions

1. Create any form.
2. Process a donation.
3. View the Donor Wall.
4. Ensure correct date is displayed on Donor Wall.
5. Toggle Show Time on/off from the inspector controls. (Verify that the date displays accordingly).
6. Toggle Show Avatar from the inspector controls. Verify the style changes match the design.

## Pre-review Checklist
